mapscript/python/tests/cases/point_test.py

```diff
@@ -70,6 +79,54 @@ def testSetXYZ(self):
             self.assertAlmostEqual(p.z, 3.0)
             self.assertAlmostEqual(p.m, 4.0)
 
+    def testPointDraw(self):
+        """Can create a point, add to a layer, and draw it directly"""
+
+        map_string = """
+        MAP
+            EXTENT 0 0 90 90
+            SIZE 500 500
+
+            SYMBOL
+            NAME "circle"
+            TYPE ELLIPSE
+            POINTS 1 1 END
+            FILLED true
+            END
+
+            LAYER
+            NAME "punkt"
+            STATUS ON
+            TYPE POINT
+            END
+
+        END"""
+
+        test_map = mapscript.fromstring(map_string)
+        layer = test_map.getLayerByName('punkt')
+        cls = mapscript.classObj()
+        style = mapscript.styleObj()
+        style.outlinecolor.setHex('#00aa00ff')
+        style.size = 10
+        style.setSymbolByName(test_map, 'circle')
+
+        cls.insertStyle(style)
+        class_idx = layer.insertClass(cls)
+        point = mapscript.pointObj(45, 45)
+
+        img = test_map.prepareImage()
+        point.draw(test_map, layer, img, class_idx, "test")
+
+        filename = 'testDrawPoint.png'
+        with open(filename, 'wb') as fh:
+            img.write(fh)
+
+        if have_image:
+            pyimg = Image.open(filename)
+            assert pyimg.size == (500, 500)
+        else:
+            assert 1
```

Comment:
assert 1 is a no-op.

Reply:
This was taken from the approach in image_test.py - if PIL isn't installed then the test passes automatically

Reply:
This is just useless code. No need to perpetuate bad past patterns :-)

Reply:
I mean: you don't need that else: assert 1 (or True) branch at all. It doesn't bring anything. Or I'm missing something...

Reply:
I thought it might result in pytest ignoring it as a test. I just checked and it will pass without the assert (assuming it gets to the end of the function) - so removing.